### PR TITLE
added voting end date to voting info page

### PR DESCRIPTION
**Pull request by kburnell**

### CODE REVIEW

**Comment by csell5** [comment]
#### 📝Changed routes:


| [**/wi/voting**](https://that-website-git-kb-voting-date.that-conference.now.sh/wi/voting) |
|:-:|
|<a href="https://that-website-git-kb-voting-date.that-conference.now.sh/wi/voting">Screenshot of /wi/voting</a><br /><sup><a href="https://images.weserv.nl?url=https://microlink-cdn.s3.us-east-1.amazonaws.com/mX2IrG7EhdP9x2dcdvFYO4PWM08_.png">(view full size)</a> |







Commit 10bc370b0bce73f4ef5e03157ee23755e7d8a805 (https://that-website-git-kb-voting-date.that-conference.now.sh).

File: pages/wi/voting.js [hunk 1]
```diff
@@ -198,6 +198,10 @@ const VotingStart = ({ user, loading }) => {
                       </strong>{' '}
                       for more info on sharing your awesome.
                     </li>
+                    <li>
+                      {/* This should not be hard coded */}
```
Comment (brettski) [inline pages/wi/voting.js]: Seriously

Comment (brettski) [reply pages/wi/voting.js]: What about using the milestone?

Comment (kburnell) [reply pages/wi/voting.js]: There is no key/id on a milestone so I would have to magic-string the text which is pretty much same as hard-coding the date.

Comment (brettski) [reply pages/wi/voting.js]: Perhaps, but it allows the date to be changed without redeploying, etc.

Comment (saragibby) [reply pages/wi/voting.js]: The ability to query for a specific milestone to get its date is something we've all talked about several times, but has yet to rise to the top of the heap in priorities. For the time being, if we want this (or any other specific milestone) date on a page it has to be hard coded.

Comment (csell5) [reply pages/wi/voting.js]: I'd suggest we just pull that out to a "constants" like file now and reference that. Then when the right time comes all we need to do is update one space regardless.

**Pull request merged**